influxdb/event_parser.go

```diff
@@ -104,7 +104,7 @@ readRow:
 	switch q.row[0] {
 	case "":
 		if len(q.row) <= 5 {
-			return errors.New("Unexpectedly few columns")
+			return errors.New("Unexpectedly few columns -- Kapacitor cannot understand this data. We need at minimum a _time column, table number, and at least one column with data in")
```

Comment:
Errors shouldn't have a period at the end, this is because they get wrapped.   I'm starting with a cap, because we already that everywhere in the code, even if it is nonstandard.  But the period at the end shouldn't be done.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Ah good point, thanks.